check_tv.py

```
@@ -609,6 +663,7 @@ def selwebdriver(live_url, timeisshit):
               check_is_live_api(new_url, config.ffmpeg, "this")
       finally:
             logging.info('edit finished contiue the stream')
+            return filenametwitch
```

Ruff (0.8.2)

`return` inside `finally` blocks cause exceptions to be silenced

(B012)